default.py

```diff
@@ -143,108 +143,82 @@ def display_nfl_network_archive(self):
-                                isPlayable = 'true'
-                                isBlackedOut = 'false'
-                                listitem.setProperty('video_id', video_id)
-                                listitem.setProperty('game_versions', 'Live')
-                        else:
-                            if weekSet == 'weekCompletedGames':
-                                if game['video']['videoId']:
-                                    video_id = str(game['video']['videoId'])
-                                    isPlayable = 'true'
-                                    isBlackedOut = 'false'
-                                    listitem.setProperty('video_id', video_id)
-
-                    listitem.setProperty('isPlayable', isPlayable)
-                    listitem.setProperty('isBlackedOut', isBlackedOut)
-                    listitem.setProperty('game_id', game_id)
-                    listitem.setProperty('away_thumb', 'http://i.nflcdn.com/static/site/7.4/img/logos/teams-matte-144x96/%s.png' % away_team)
-                    listitem.setProperty('home_thumb', 'http://i.nflcdn.com/static/site/7.4/img/logos/teams-matte-144x96/%s.png' % home_team)
-                    #listitem.setProperty('game_date', game['date'].split('T')[0])
-                    #listitem.setProperty('game_versions', ' '.join(game_versions))
-                    self.games_items.append(listitem)
-            self.games_list.addItems(self.games_items)
+                        from_zone = tz.tzutc()
+                        to_zone = tz.tzlocal()
+                        game_datetime = datetime.datetime(*(time.strptime(game['gameDateTimeUtc'], date_time_format)[0:6]))
+                        game_datetime = game_datetime.replace(tzinfo=from_zone)
+                        local_time = game_datetime.astimezone(to_zone)
+                        if addon.getSetting('local_tz') == '1':  # localize and use 12-hour clock
+                            game_info = local_time.strftime('%A, %b %d - %I:%M %p')
+                        else:  # localize and use 24-hour clock
+                            game_info = local_time.strftime('%A, %b %d - %H:%M')
+                except:  # all else fails, just use their raw date value
+                    game_datetime = game['gameDateTimeUtc'].split('T')
+                    game_info = game_datetime[0] + '[CR]' + game_datetime[1].split('.')[0] + ' ET'
+
+            if game['videoStatus'] == 'SCHEDULED':
+                isPlayable = 'false'
+                isBlackedOut = 'false'
+            elif game['videoStatus'] == 'LIVE':
+                game_info += '[CR]» Live «'
+                video_id = str(game['video']['videoId'])
+                isPlayable = 'true'
+                isBlackedOut = 'false'
+                listitem.setProperty('video_id', video_id)
+                listitem.setProperty('game_versions', 'Live')
+            else:  # ONDEMAND
```

Comment:
Basically, all games that are archived has the 'videoStatus' set to 'ONDEMAND'. There's probably some additional info we're going to need in regards to blacked out games etc, but this should be easy to add later on.